index.html

```diff
@@ -662,9 +662,11 @@ <h3>Accessible Name Calculation</h3>
 				<dd>One of the following values:
 					<ol>
 						<li>author: name comes from values provided by the author in explicit markup features such as the <pref>aria-label</pref> attribute, the <pref>aria-labelledby</pref> attribute, or the host language labeling mechanism, such as the <code>alt</code> or <code>title</code> attributes in HTML, with HTML <code>title</code> attribute having the lowest precedence for specifying a text alternative.</li>
+						<li>heading: name comes from the text value of the first descendant (i.e., depth first) <a>element</a> node with the role of <code>heading</code>. Although "heading" is allowed in addition to "author" in some <a>roles</a>, "heading" is used in content only if higher priority "author" features are not provided.</li>
```

Comment:
Just to clarify, does this mean:
- Take the name from the first descendant that's a heading?
- Take the name from the the first descendant if it's a heading?

Should we ignore anything in the depth-first search, such as hidden, generic or role="none"|"presentation" elements?

Reply:
The first matching... which you phrased above as "Take the name from the first descendant that's a heading?"

I thought that was clear from the existing text, but obvs open to word-smithing the PR if it improves the clarity.

Reply:
```suggestion
						<li>heading: name comes from the text value of the first descendant <a>element</a> node matching the role of <code>heading</code> in an <em>iterative deepening depth-first search.</em> Although "heading" is allowed in addition to "author" in some <a>roles</a>, "heading" is used in content only if higher priority "author" features are not provided.</li>
```

Reply:
Looks clear to me now.